Source/Core/Core/HW/EXI/EXI_DeviceIPL.h

```diff
@@ -42,8 +42,10 @@ class CEXIIPL : public IEXIDevice
   {
     ROM_BASE = 0,
     ROM_SIZE = 0x200000,
-    SRAM_BASE = 0x800000,
-    SRAM_SIZE = 0x44,
+    RTC_BASE = 0x800000,
+    RTC_SIZE = 0x3,
```

Comment:
Is the RTC really just 3 bytes instead of 4? YAGCD says 32-bit value as well.

Reply:
it is indeed 4, starting at 0x800000 to 0x800003, 0x800004 is sram. if i make the lenght 4, it'll be 5 bytes inc 0x800004 will it not?

Reply:
Uh, why? `0x800000` for `0x4` items is `0x800000`, `0x800001`, `0x800002` and `0x800003`. Or am I misunderstanding what you mean?

Reply:
ye no, youre right. its odd that the gamecube IPL could display the correct time then if it technically read a byte to little. 
then again, thats probably cause it does a 4 length read from 0x800000 and not 0x800000,0x800001,0x800002 etc etc , in which the address would get relatched